app/src/main/res/values-fr/strings.xml

```diff
@@ -121,6 +120,8 @@
     <string name="uploadProgressDescriptionTemplateConfidentiality">Nous sommes indépendants et contribuons à %s</string>
     <string name="uploadProgressDescriptionTemplateEnergy">Nous concevons %s, pour dépasser les standards d’efficacité énergétique du marché.</string>
     <string name="uploadProgressDescriptionTemplateIndependence">Nous développons %s Sans compromis sur l’écologie, la vie privée et l’humain.</string>
+    <string name="uploadErrorTitle">Qu’est ce que quoi ?</string>
+    <string name="uploadErrorDescription">Le transfert n’a pas fonctionné, donnes-lui une autre chance. Si le problème persiste, contacte notre support.</string>
```

Comment:
Shouldn't this string be before the previous one ?

It's not the 1st time that in French _(and not the other locales)_, strings aren't alphabetically ordered in this project, I don't understand why.

Reply:
Hmm interesting question. I know I've had merge conflicts on these lines and had to manually select these two to solve the conflict which means that I could've missclicked their order. But if you say that this is a common occurence, maybe the import script could be broken? 

Now I want to actually parse XML even more so everything works perfectly well in the script ...

Reply:
It's the 2nd time I see this issue.
But maybe the 1st time it was also a simple missclick that changed the order.

Let's wait & see if it happens a third time.